### PR TITLE
FIX: Emoji sets should contain all available emojis for the set.

### PR DESCRIPTION
**Pull request by tgxworld**
* Emojis not available in a particular set is simply not loaded.

### CODE REVIEW

File: lib/tasks/emoji.rake [hunk 1]
```diff
@@ -113,7 +114,13 @@ def write_emojis(emojis, aliases, style, folder)
...
 def write_emoji(path, emoji)
-  open(path, "wb") { |f| f << emoji }
-  `pngout #{path}`
-  putc "."
+  begin
+    open(path, "wb") { |f| f << emoji }
+    `pngout #{path}`
+    putc "."
+  ensure
+    if !File.size?(path)
```
Comment (ZogStriP) [inline lib/tasks/emoji.rake]: Why not 

```ruby
raise "Failed to write emoji: #{path}" unless File.size?(path)
```

Comment (tgxworld) [inline lib/tasks/emoji.rake]: `unless` throws me off most of the time haha

Comment (ZogStriP) [inline lib/tasks/emoji.rake]:
```ruby
raise "Failed to write emoji: #{path}" if !File.size?(path)
```

is fine too

Comment (tgxworld) [inline lib/tasks/emoji.rake]: O do you mean why not one line it?

**Comment by tgxworld** [comment]
Closing this first since we're only going to support the intersection of emojis.

**Pull request closed without merging**